pyscf/lib/libqcschema.py

```diff
@@ -0,0 +1,256 @@
+'''
+This lib loads results that are in a QCSchema format json.
+Funcs below can recreate the mol and scf objects from the info in the json.
+'''
+import json
+import numpy as np
+import pyscf
+from pyscf.lib.parameters import BOHR
+
+def load_qcschema_json( file_name ):
+    '''
+    Does: loads qcschema format json into a dictionary
+    Input:
+        file_name: qcschema format json file
+
+    Returns: dict in qcschema format
+    '''
+    # load qcschema output json file
+    data = None
+    with open(file_name,'r') as f:
+        data = json.load(f)
+    return data
+
+def load_qcschema_go_final_json( file_name ):
+    '''
+    Does: loads qcschema format geometry optimization json 
+          and returns only the optimized 'final' geometry 
+          qcschema info as a dictionary. 
+          
+    Input:
+        file_name: qcschema format json file
+
+    Returns: dict in qcschema format
+    '''
+    # load qcschema GO output json file 
+    # and return last 'trajectory' point's entries
+    # (this is the optimized molecule)
+    data = None
+    temp = None
+    with open(file_name,'r') as f:
+        temp = json.load(f)
+    data = temp["trajectory"][-1]
+    return data
+
+def load_qcschema_molecule(qcschema_dict, to_Angstrom=False, xyz=False, mol_select=1, step=0):
```

Comment:
what is `step`?

Reply:
for geometry optimization or a trajectory, qcschema outputs have info about each geometry. The 'step' var allows one to access the info about that step/geometry. Added this in the comments:
       
       step: for geometry optimization or trajectory, which have multiple molecules in a qcschema output.
              This specifies which step to load the molecule from.